forge/test/model_demos/high_prio/cnn/pytorch/test_blazepose.py

```diff
@@ -28,8 +28,12 @@ def test_blazepose_detector_pytorch(test_device):
 
     # Load BlazePose Detector
     pose_detector = BlazePose()
-    pose_detector.load_weights("mediapipepytorch/blazepose.pth")
-    pose_detector.load_anchors("mediapipepytorch/anchors_pose.npy")
+    pose_detector.load_weights(
```

Comment:
I don't think we'll be able to load Blaze Pose from the CCM repo at the moment. 

Let me double-check if we have a better mechanism to handle this type of input. In the meantime, let's do the following:
- Add a note here and run this model without weights
- See if there is a model version that has the option to download and cache weights. Maybe it'll be a good option to define this logic on our side. E.g. function to download and cache model weights from certain URLs, similarly to how HuggingFace is caching their weights (as part of .cache/* dir)

Reply:
I send a request to get some sort of storage for these .pth, and future ONNX files. I'm not sure when we're going to get it, so let's put this variant on pause if there isn't any other suitable.